### PR TITLE
Add 'Codecademy' website presence

### LINKED ISSUE #1830: Codecademy | www.codecademy.com
**Prerequisites and essential questions** 
- [x] Is it a popular site? || Yes
- [x] Is the website older than 2 months? || Yes
- [ ] Is the site locked to a specific country/region? || No
- [ ] Is the site a paid service? (e.g. Netflix, Hulu) || No
- [ ] Does the website feature NSFW content? (e.g. porn, etc...) || No
- [ ] Are you a donator/patron? || No
- [x] Do you acknowledge that coding presences is completely voluntary and may take time for your service to be added regardless of priority? || Yes

♫ ฅ^✯-✯^ฅ ♪#3526

https://www.codecademy.com/

The presence would display as an image : the logo of codecademy
The name (js, py, json etc)
The lesson (variables, loops etc)
Time passed

[Image](https://res-4.cloudinary.com/crunchbase-production/image/upload/c_lpad,f_auto,q_auto:eco/veuzby04rzsqwpk4d6wz)

### PR DESCRIPTION
**Pull request by d0gg3r**
Resolves #1830 (and #1889)

image

image

image

image

image

**And support for many more endpoints!**

### CODE REVIEW

File: websites/C/Codecademy/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "id": "460318444487704597"
+  },
+  "url": ["www.codecademy.com", "news.codecademy.com"],
+  "description": {
+    "en": "Learn the technical skills you need for the job you want. As leaders in online education and learning to code, we’ve taught over 45 million people using a tested curriculum and an interactive learning environment. Start with HTML, CSS, JavaScript, SQL, Python, Data Science, and more."
+  },
+  "service": "Codecademy",
+  "version": "1.1.2",
+  "logo": "https://i.imgur.com/TnKcxoz.jpg",
+  "thumbnail": "https://i.imgur.com/5rpD3o0.png",
```
Comment (frozaum) [inline websites/C/Codecademy/dist/metadata.json:14]: Please change the thumbnail to a promotional image or a screenshot of the website.

**Comment by doomlerd** [comment]
Update your fork

**Comment by d0gg3r** [comment]
Still failing with `reviewdog: environment variable $REVIEWDOG_GITHUB_API_TOKEN is not set`.

File: websites/C/Codecademy/dist/metadata.json (hunk 1 above)
Comment (d0gg3r) [reply websites/C/Codecademy/dist/metadata.json]: Resolved in fd75ba5.

**Comment by d0gg3r** [comment]
Now failing with `reviewdog: cannot get repo owner from environment variable. Set CI_REPO_OWNER?`

**Comment by frozaum** [comment]
Hello, as of right now, we won't be able to review your pull request due to the temporarily locking of this repository that we announced in #1946, so we gladly request your patience until everything is sorted out. :)

**Comment by frozaum** [comment]
Hello, as of right now, we won't be able to review your pull request due to the temporarily locking of this repository that we announced in #1946, so we gladly request your patience until everything is sorted out. :)

**Comment by Bas950** [comment]
Unlocked (No longer on hold) - Please update your fork to get your Pull Request reviewed, if you don't do it within the next 3 days the Pull Request will be closed.

**Comment by d0gg3r** [comment]
Will do today.

**Comment by d0gg3r** [comment]
It's failing with the review dog api not set or something... I did git rebase so not sure what I did wrong

File: websites/C/Codecademy/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "author": {
+    "name": "d0gg3r",
+    "id": "460318444487704597"
+  },
+  "url": ["www.codecademy.com", "news.codecademy.com"],
+  "description": {
+    "en": "Learn the technical skills you need for the job you want. As leaders in online education and learning to code, we’ve taught over 45 million people using a tested curriculum and an interactive learning environment. Start with HTML, CSS, JavaScript, SQL, Python, Data Science, and more."
+  },
+  "service": "Codecademy",
+  "version": "1.1.3",
```
Comment (Bas950) [inline websites/C/Codecademy/dist/metadata.json:12]: version 1.0.0 on first release

File: websites/D/Discord/dist/metadata.json [hunk 3]
```diff
@@ -35,4 +35,4 @@
     "voice"
   ],
   "category": "socials"
-}
\ No newline at end of file
+}
```
Comment (Bas950) [inline websites/D/Discord/dist/metadata.json]: undo

**Comment by doomlerd** [comment]
Closing in 2 days.

File: websites/C/Codecademy/dist/metadata.json (hunk 2 above)
Comment (d0gg3r) [reply websites/C/Codecademy/dist/metadata.json]: Resolved in 41be2e3.

File: websites/D/Discord/dist/metadata.json (hunk 3 above)
Comment (d0gg3r) [reply websites/D/Discord/dist/metadata.json]: Resolved in e37ee76.

**Review by Bas950 — APPROVED** [review]

**Pull request merged**